tests/test_application.py

```diff
@@ -267,3 +268,31 @@ def helper(scopes, account, authority, *args, **kwargs):
         with self.assertRaises(ExpectedBehavior):
             self.app.acquire_token_silent(["different scope"], self.account)
 
+
+class TestApplicationForClientCapabilities(unittest.TestCase):
+
+    def test_merged_claims_returned_correctly(self):
+        client_capabilities = ["llt", "ssm"]
+        claims = "{\"id_token\": {\"auth_time\": {\"essential\": true}}}"
```

Comment:
Perhaps it is time to utilize Python's triple-quotation string? You can also use it to break down those long lines.
```suggestion
        claims = '''{"id_token": {"auth_time": {"essential": true}}}'''
```